Doc/c-api/object.rst

```diff
@@ -575,3 +575,23 @@ Object Protocol
    has the :c:macro:`Py_TPFLAGS_MANAGED_DICT` flag set.
 
    .. versionadded:: 3.13
+
+.. c:function:: int PyUnstable_Object_EnableDeferredRefcount(PyObject *obj)
+
+   Enable `deferred reference counting <https://peps.python.org/pep-0703/#deferred-reference-counting>`_ on *obj*,
+   if supported by the runtime.  In the :term:`free-threaded <free threading>` build,
+   this allows the interpreter to avoid reference count adjustments to *obj*,
+   which may improve multi-threaded performance.  The tradeoff is
+   that *obj* will only be deallocated by the tracing garbage collector.
+
+   This function returns ``1`` if deferred reference counting was enabled on *obj*,
+   and ``0`` if deferred reference counting is not supported or if the hint was
+   ignored by the runtime. This function is thread-safe, and cannot fail.
+
+   This function is a no-op on builds with the :term:`GIL` enabled, which do
+   not support deferred reference counting. This also no-ops if *obj* is not
+   an object tracked by the garbage collector (see :func:`gc.is_tracked` and
```

Comment:
I don't think I've ever seen “no-ops” as a verb; would this be easier to read?

```suggestion
   This function does nothing on builds with the :term:`GIL` enabled, which do
   not support deferred reference counting. This also does nothing if *obj* is not
   an object tracked by the garbage collector (see :func:`gc.is_tracked` and
```
(Though I do enjoy no-oping, myself!)

Reply:
I wish it was a verb, it's fun to say :)